apps/envConstants.js

```diff
@@ -13,8 +13,14 @@ function getBoolEnv(name) {
 function getMoocEnv(name) {
   var moocName = 'MOOC_' + name;
   if (process.env.moocName) {
-    console.warn('The ' + moocName + ' environment variable is deprecated. ' +
-      'Use ' + name + ' instead.');
+    console.warn(
```

Comment:
You can use string interpolation to compress this into one line.

Reply:
Why was this file modified at all?

Reply:
I merged in staging and it asked me to resolve a prettier error - I didn't mean to check it in though. Will remove.